examples/gnrc_border_router/Makefile

```diff
@@ -62,6 +62,14 @@ else ifeq (uhcp,$(PREFIX_CONF))
   USEMODULE += gnrc_uhcpc
 else ifeq (auto_subnets,$(PREFIX_CONF))
   USEMODULE += gnrc_ipv6_auto_subnets_simple
+else ifeq (static,$(PREFIX_CONF))
+  IPV6_ADDR ?= 2001:db8:1::1
+  IPV6_DEFAULT_ROUTER ?= fe80::1
+
+  CFLAGS += -DGNRC_IPV6_STATIC_ADDR=\"$(IPV6_ADDR)\"
+  CFLAGS += -DGNRC_IPV6_STATIC_PREFIX=\"$(IPV6_PREFIX)\"
+  CFLAGS += -DGNRC_IPV6_STATIC_DEFAULT_ROUTER=\"$(IPV6_DEFAULT_ROUTER)\"
```

Comment:
Would it make sense to use Kconfig here instead? (and also maybe make @MrKevinWeiss's job a bit easier)